src/util/pelz_io.c

```diff
@@ -84,34 +83,38 @@ int get_file_ext(charbuf buf, int *ext)
     {
       if (memcmp(buf.chars + period_index, ext_type[i], strlen(ext_type[i])) == 0)
       {
-        *ext = i + 1;
-        break;
+        return (i + 1);
```

Comment:
I'd rather explicitly return the type, rather than counting on the ordering in the enum being right.

Reply:
Explicitly returns the type now